spec/index.bs

```diff
@@ -8135,112 +8135,112 @@ Issue: Because timestamp query provides high-resolution GPU timestamp, we need t
 
 ## {{HTMLCanvasElement/getContext()|HTMLCanvasElement.getContext()}} ## {#canvas-getcontext}
 
-A {{GPUPresentationContext}} object can be obtained via the {{HTMLCanvasElement/getContext()}}
-method of an {{HTMLCanvasElement}} instance by passing the string literal `'gpupresent'` as its
+A {{GPUCanvasContext}} object can be obtained via the {{HTMLCanvasElement/getContext()}}
+method of an {{HTMLCanvasElement}} instance by passing the string literal `'webgpu'` as its
 `contextType` argument.
 
 <div class="example">
-    Get a {{GPUPresentationContext}} from an offscreen {{HTMLCanvasElement}}:
+    Get a {{GPUCanvasContext}} from an offscreen {{HTMLCanvasElement}}:
     <pre highlight="js">
         const canvas = document.createElement('canvas');
-        const context = canvas.getContext('gpupresent');
+        const context = canvas.getContext('webgpu');
         context.configure(/* ... */);
         // ...
     </pre>
 </div>
 
-## GPUPresentationContext ## {#presentation-context}
+## GPUCanvasContext ## {#canvas-context}
 
 <script type=idl>
 [Exposed=(Window, DedicatedWorker)]
-interface GPUPresentationContext {
+interface GPUCanvasContext {
     readonly attribute (HTMLCanvasElement or OffscreenCanvas) canvas;
 
-    undefined configure(GPUPresentationConfiguration configuration);
+    undefined configure(GPUContextConfiguration configuration);
```

Comment:
maybe `GPUCanvasConfiguration`?

Reply:
Done! I actually was going to go with `CanvasConfiguration` originally but second guessed myself. I can see how `ContextConfiguration` would be ambiguous, though.